src/lcd.cpp

```diff
@@ -546,7 +546,7 @@ void LcdTask::displayInfoLine(LcdInfoLine line, unsigned long &nextUpdate)
       gettimeofday(&local_time, NULL);
       struct tm timeinfo;
       localtime_r(&local_time.tv_sec, &timeinfo);
-      strftime(temp, sizeof(temp), "%d/%m/%Y", &timeinfo);
```

Comment:
I think this is more of a personal taste/country specific convention, so should probably handle separately, probably needs to be user configurable.